src/main/java/com/google/codeu/data/Location.java

```diff
@@ -5,18 +5,23 @@
 public class Location {
     public String name;
     public String description;
+    public Double lat;
```

Comment:
These should be private. (I know that the name and description are public - those should be private too, but we can deal with them later.)